### DISCUSSION TITLE
showpage desatived

### DISCUSSION THREAD

**Discussion by jc101b** [ImageMagick/ImageMagick#7352] [Development]
### ImageMagick version

7.1.1-33

### Operating system, version and so on

windows

### Description

helo...
I don't want the "showpage" command to be printed in the .ps.
I would have a similar option to this:
"J:\JCexe\ImageMagick\magick.exe"  -noshowpage w06.jpg w06.ps

**Comment by urban-warrior** [comment]
Try this command:

```
Try this command:

```
"J:\JCexe\ImageMagick\magick.exe" w06.jpg eps:w06.ps
```

**Reply by fmw42** [reply]
I do not see that option -noshowpage on the list of methods on the options page.  Please add and explain what it does.

**Reply by urban-warrior** [reply]
Cut/paste error.  Fixed.

**Reply by jc101b** [reply]
> I do not see that option -noshowpage on the list of methods on the options page. Please add and explain what it does.
I don't want the "showpage" command to be printed in the .ps
How do I prevent it from being printed in the .ps file that is generated?

**Reply by jc101b** [reply]
> Try this command:
> 
> ```
> Try this command:
> ```
> 
> "J:\JCexe\ImageMagick\magick.exe" w06.jpg eps:w06.ps

j:\Sci\gs>"J:\JCexe\ImageMagick\magick.exe" w06.jpg eps:w06.ps
magick.exe: RegistryKeyLookupFailed `CoderModulesPath' @ error/module.c/GetMagickModulePath/668.
magick.exe: no decode delegate for this image format `JPEG' @ error/constitute.c/ReadImage/746.
j:\Sci\gs>

**Reply by jc101b** [reply]
> I do not see that option -noshowpage on the list of methods on the options page. Please add and explain what it does.
part of the generated .ps file

/DisplayImage
{
  %part of the generated .ps file
  % Display a DirectClass or PseudoClass image.
  %
  % Parameters:
  %   x & y translation.
  %   x & y scale.
  %   label pointsize.
  %   image label.
  %   image columns & rows.
  %   class: 0-DirectClass or 1-PseudoClass.
  %   compression: 0-none or 1-RunlengthEncoded.
  %   hex color packets.
  %
  gsave
  /buffer 512 string def
  /byte 1 string def
  /color_packet 3 string def
  /pixels 768 string def

  currentfile buffer readline pop
  token pop /x exch def
  token pop /y exch def pop
  x y translate
  currentfile buffer readline pop
  token pop /x exch def
  token pop /y exch def pop
  currentfile buffer readline pop
  token pop /pointsize exch def pop
  /Times-Roman findfont pointsize scalefont setfont
  /label 512 string def
  currentfile label readline pop
  x y 12 add moveto label show pop
  x y scale
  currentfile buffer readline pop
  token pop /columns exch def
  token pop /rows exch def pop
  currentfile buffer readline pop
  token pop /class exch def pop
  currentfile buffer readline pop
  token pop /compression exch def pop
  class 0 gt { PseudoClassImage } { DirectClassImage } ifelse
  grestore
  **showpage**  % _<<<<--this line with this instruction that I don't want to appear_
} bind def
%%EndProlog
%%Page:  1 1
%%PageBoundingBox: 0 0 768 1024
DisplayImage
0 0
768 1024
12
768 1024
0
0

**Comment by jc101b** [comment]
Helo....
Just to clarify why this solution of disabling the "showpage" is that when I generate a .pdf with four images, if it is in the .ps the "showpage" line then the ghostscript generates a pdf page per image.
If I delete the "showpage" line then I can get the 4 images on the same PDF page.
From what I understand, "showpage", if defined in the .ps, ends the creation of the pdf. So I leave the line in the last .ps to be included.

**Comment by fmw42** [comment]
I do not know much about showpage, but also see Imagemagick command -+adjoin at https://imagemagick.org/script/command-line-options.php#adjoin
